test(napi/parser): disable raw transfer tests on `antd.js` fixture

overlookmotel opened the pull request:
The raw transfer test on `antd.js` fixture is timing out sometimes on CI. We already disabled the "range & parent" test on this fixture. Disable the other test on it too. But only skip these tests in CI.

(alternative to #14438)

camc314 reviewed (approved):
two nits, feel free to ignore

overlookmotel committed:
test(napi/parser): disable raw transfer tests on `antd.js` fixture (#14446)

The raw transfer test on `antd.js` fixture is timing out sometimes on CI. We already disabled the "range & parent" test on this fixture. Disable the other test on it too. But only skip these tests in CI.

(alternative to #14438)
